### DISCUSSION TITLE
robots.txt generation

### DISCUSSION THREAD

**Discussion by NyasakiAT** [statiqdev/Discussions#54] [Q&A]
Hi, is there a way to generate the robots.txt?
Since i can't upload one because im using Netlify

**Comment by TheBrambleShark** [comment]
What's stopping you from writing one by hand and putting it in the root of your input folder?
.txt files are copied directly to the output folder, so it should end up there right next to your sitemap.xml

![output](https://user-images.githubusercontent.com/7003081/108587603-802be080-734c-11eb-846e-abd421c44ec0.png)

**Reply by daveaglick** [reply]
This is probably the best way to do it. We could consider adding a feature where robots settings are added to document front matter and the robots.txt file is generated from that (please add an issue if that would be helpful), but the most direct is probably creating the file manually with the correct settings.
